### PR TITLE
bpo-31937: Add the term "dunder" to the glossary

### PR DESCRIPTION
**Pull request by matrixise**
https://bugs.python.org/issue31937

### CODE REVIEW

File: Doc/glossary.rst [hunk 1]
```diff
@@ -299,6 +299,11 @@ Glossary
       with :term:`abstract base classes <abstract base class>`.)  Instead, it
       typically employs :func:`hasattr` tests or :term:`EAFP` programming.
 
+   dunder
+      A dunder is a variable, method or package whose name is with two leading
```
Comment (Mariatta) [inline Doc/glossary.rst:303]: We should say that it's short for "double underscore".

**Review by Mariatta — CHANGES_REQUESTED** [review]

File: Doc/glossary.rst (hunk 1 above)
Comment (matrixise) [reply Doc/glossary.rst:303]:
```
dunder
   A dunder is a variable, method or package whose name is with two double underscore
```
?

Comment (matrixise) [reply Doc/glossary.rst:303]: or just `dunder = double underscore` ?

**Comment by Mariatta** [comment]
Sorry @matrixise, the issue has been closed and rejected. So I'll close your PR.

**Pull request closed without merging**